src/strgen/strgen.cpp

```diff
@@ -250,8 +250,8 @@ struct HeaderFileWriter : HeaderWriter, FileWriter {
 	{
 		/* Find the plural form with the most amount of cases. */
 		int max_plural_forms = 0;
-		for (uint i = 0; i < lengthof(_plural_forms); i++) {
-			max_plural_forms = std::max(max_plural_forms, _plural_forms[i].plural_count);
+		for (const auto &pf : _plural_forms) {
+			max_plural_forms = std::max(max_plural_forms, pf.plural_count);
 		}
```

Comment:
`std::max_element` ? Might be more complicated than it's worth

Reply:
`std::max_element` would work if it wasn't the `plural_count` member that we're after.

As such it would need a custom comparator, and indeed would look more complicated.

Reply:
Though

```c++
int max_plural_forms = std::max_element(std::begin(_plural_forms), std::end(_plural_forms), [](const auto &a, const auto &b) { return a.plural_count < b.plural_count; })->plural_count;
```

would do it.

Reply:
Like I say, may be more complicated than it's worth